core/DataAccess/ArchiveSelector.php

```diff
@@ -45,7 +45,7 @@ private static function getModel()
         return new Model();
     }
 
-    public static function getArchiveIdAndVisits(ArchiveProcessor\Parameters $params, $minDatetimeArchiveProcessedUTC)
+    public static function getArchiveIdAndVisits(ArchiveProcessor\Parameters $params)
```

Comment:
@diosmosis some usages still seem to pass two parameters here, eg Advertising, CustomReports, SearchEngineKeywordsPerformance, ... might be better to still keep the 2nd parameter but simply not use it. Otherwise it could break things and lead to notices etc. In Matomo 4 we got remove the 2nd parameter

Reply:
👍